### ISSUE TITLE
Select features in the map using keyboard navigation

### ISSUE THREAD

**Issue by magnusnil** [geosolutions-it/MapStore2#6917]
## Description
It should be possible to select features in the map (such as points, lines and polygons) using keyboard navigation. This does not seem possible at the moment.

Tested in Chrome and Microsoft Edge, using MapStore 2021.01.03.
